Automattic/wp-calypso
# Billing History: show tax

#### Changes proposed in this Pull Request

This PR adds and renders placeholder tax data for the billing history pages:

![billing_history_ _wordpress_com](https://user-images.githubusercontent.com/5952255/49367006-8404c580-f736-11e8-9a18-9deadf756661.jpg)

![billing_history_ _wordpress_com](https://user-images.githubusercontent.com/5952255/49366751-d1346780-f735-11e8-96df-17c403152b6d.jpg)

#### Testing instructions

Navigate to http://calypso.localhost:3000/me/purchases/billing?flags=show-tax and http://calypso.localhost:3000/me/purchases/billing and confirm that the billing history renders with and without tax details respectively, with no errors or visual issues.

Note that the placeholder tax is injected when the transaction is loaded, so if there's no tax even with the config flag enabled, try a different receipt and/or reloading with sympathy to avoid serializing the old value:
`ENABLE_FEATURES=force-sympathy npm start`
